homeassistant/components/digital_ocean/constants.py

```diff
@@ -0,0 +1,17 @@
+"""Store constants used across the integration."""
+
+from datetime import timedelta
+
+ATTR_DOMAIN_NAME = "domain"
+ATTR_RECORD_NAME = "record"
+ATTR_RECORD_VALUE = "value"
+ATTR_RECORD_TYPE = "type"
+
+# Domain validation Regex
+# from https://stackoverflow.com/questions/3026957/how-to-validate-a-domain-name-using-regex-php/16491074#16491074
+DOMAIN_NAME_REGEX = r"^(?!\-)(?:(?:[a-zA-Z\d][a-zA-Z\d\-]{0,61})?[a-zA-Z\d]\.){1,126}(?!\d+)[a-zA-Z\d]{1,63}$"
+
+# Regular expression to match IPV4 addresses
+IPV4_REGEX = r"^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$"
```

Comment:
Instead of a regex, please consider using `ipaddress.ip_address`, see for example https://github.com/home-assistant/core/blob/b9f0145b8f5aa849eeec8a27acaf0ca88cf1039d/homeassistant/components/homekit/util.py#L139

Reply:
Did you look into this, @phrfpeixoto ?

Reply:
Yes.